client.go

```diff
@@ -184,6 +185,23 @@ func (c *Client) Embed(model string, opts EmbedOptions) (*EmbedResponse, error)
 	return ret, nil
 }
 
+// Extracts entities of specified types from the provided text. Each extraction
+// contains a type and a confidence score.
+// See: https://docs.cohere.ai/extract-reference
+// Returns an EmbedResponse object.
```

Comment:
Is this true?

Reply:
- Re: confidence score. Not true yet, but that's what I'm currently working on. 
- EmbedResponse object, not true will fix